deepchecks/utils/wandb_utils.py

```diff
@@ -0,0 +1,50 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Wandb utils module."""
+from typing import Any
+
+try:
+    import wandb
+
+    assert hasattr(wandb, '__version__')  # verify package import not local dir
+except (ImportError, AssertionError):
+    wandb = None
+
+__all__ = ['wandb_init_helper']
+
+
+def wandb_init_helper(dedicated_run: bool, default_config: dict, **kwargs: Any):
```

Comment:
default values for default_config and dedicated_run, looks like they aren't mandatory even if you always set them.

Reply:
and maybe rename to something else? it's confusing what is an `init_helper`? 
suggestion: `set_wandb_run_state`

Reply:
wdym the default values?

Reply:
I agree with @benisraeldan about the function name